administrator/components/com_media/resources/components/app.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div>
+        <div class="row-fluid">
+            <div class="span3 media-sidebar">
+                <media-tree :tree="tree" :dir="dir"></media-tree>
+            </div>
+            <div class="span9 media-browser">
+                <media-browser :content="content"></media-browser>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    export default {
+        name: 'media-app',
+        data() {
+            return {
+                // The current selected directory
+                dir: '/',
+                // The content of the selected directory
+                content: [],
+                // The tree structure
+                tree: {path: '/', children: []},
+                // The api base url
+                baseUrl: 'https://api.github.com/repos/joomla/joomla-cms/contents'
+            }
+        },
+        methods: {
+            getContent() {
+                let url = this.baseUrl + this.dir;
+                jQuery.getJSON(url, (content) => {
+                    // Update the current directory content
+                    this.content = content;
+                    // Find the directory node by path and update its children
+                    this._updateLeafByPath(this.tree, this.dir, content);
+                }).error(() => {
+                    alert("Error loading directory content.");
+                })
```

Comment:
This is not the final solution for notifications & error handling. Its just a placeholder.
We have to implement vue notifications in a more abstract way, so that we can use it across components, services and views.

For multilang support we will use something like https://github.com/kazupon/vue-i18n and use the default joomla language files.